src/dryrun.ts

```diff
@@ -139,7 +139,7 @@ export async function createDryrun({
   await Promise.all(acctPromises);
 
   return new DryrunRequest({
-    txns: txns.map((st) => ({ ...st, txn: st.txn.get_obj_for_encoding() })),
```

Comment:
Not really, that's more how the old code work, where `get_obj_for_encoding` was implemented generically by `BaseModel` and it attempted to call `get_obj_for_encoding` on all field values. See this file from develop: https://github.com/algorand/js-algorand-sdk/blob/981905ad8511cea706fc77d0c133bb1962f76927/src/client/v2/basemodel.ts

With the changes in this PR, it's more like `DryrunRequest` knows which of its fields are `Encodable` and it will call `toEncodingData` on them when its own `toEncodingData` method is called. There is no generic searching all objects for the magic function name.